handler_syslog.go

```diff
@@ -76,6 +76,8 @@ func (self *SyslogHandler) Emit(record *LogRecord) error {
 		err = self.writer.Info(message)
 	case LevelDebug:
 		err = self.writer.Debug(message)
+	case LevelTrace:
+		err = self.writer.Debug(message)
```

Comment:
Will it be better if we use the default branch for `LevelTrace`?

Reply:
Moved to the default branch in commit a9171f8